api/server/response_options.proto

```diff
@@ -5,6 +5,16 @@ package nighthawk.server;
 import "google/protobuf/wrappers.proto";
 import "validate/validate.proto";
 import "envoy/api/v2/core/base.proto";
+import "google/protobuf/duration.proto";
+
+message ConcurrencyBasedDelay {
+  // Minimal delay to add to replies.
+  google.protobuf.Duration minimal_delay = 1 [(validate.rules).duration.gte.nanos = 0];
+  // Factor to use when adding latency as concurrency increases.
+  google.protobuf.Duration concurrency_delay_factor = 2 [(validate.rules).duration.gte.nanos = 0];
+  // Multiplier.
+  uint64 concurrency_delay_multiplier = 3;
```

Comment:
This appears to be unused.

Reply:
good catch, that needs to go indeed. FYI, I'm currently working to add tests and docs, to get this PR in shape